src/Http/Routing/test/UnitTests/Matching/DfaMatcherBuilderTest.cs

```diff
@@ -231,7 +231,9 @@ public void BuildDfaTree_MultipleEndpoint_LiteralDifferentCase()
             Assert.Null(root.Parameters);
 
             var next = Assert.Single(root.Literals);
-            Assert.Equal("a", next.Key);
+            // Ignore case in 2.2 because EndpointComparer.ComparePattern does not specify a culture and running in the wrong culture
+            // will change the order of results. This causes 'A' to be used instead of 'a'
+            Assert.Equal("a", next.Key, ignoreCase: true);
```

Comment:
Is this really limited to just **this** line/comparison?

Reply:
I'm not completely sure because I can't replicate it on my machine. If it isn't then we'll start to get new failures later in the test.

Reply:
Ah I see from looking at a bit more context. This might be enough. 

It looks like the `b1` and `b2` is already sort order independent as well.